pages/[slug]/index.js

```diff
@@ -64,7 +64,8 @@ const Page = ({ slug, data, html, locales }) => {
             backgroundColor: 'background',
             svg: {
               mr: [0, 2]
-            }
+            },
+            py: ['20px', 2]
```

Comment:
```suggestion
            py: ['15px', 2]
```

Makes a nice round button!
![Screen Shot 2022-04-18 at 4 29 36 PM](https://user-images.githubusercontent.com/621904/163873005-d41a3796-9acf-4ec8-a2de-189e6dcf2d32.png)

Reply:
I've achieved a perfect button with `14px` of vertical padding 🙇

<img width="228" alt="Screen Shot 2022-04-18 at 5 30 37 PM" src="https://user-images.githubusercontent.com/34525547/163881055-a7bafa4e-3ba1-4b63-8ab1-6f9c3c1d64ee.png">